controllers/details.js

```diff
@@ -1,21 +1,23 @@
 'use strict';
 
-var api = require('../api');
-var helpers = require('../lib/helpers');
+const api = require('../api');
+const helpers = require('../lib/helpers');
+const fields = require('../fields');
+const _ = require('underscore');
 
 module.exports = function renderDetails(req, res) {
   if (req.params && req.params.sysnum) {
     const username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
     const canRedirectToResults = (req.query && req.query.multipleResults) !== undefined;
+
```

Comment:
Meh, I like to separate out the var / const declarations.

I sometimes think the HMPO style of multiple things per var makes more sense.